src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1058,6 +1058,40 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('lease_duration', type=int)
         c.argument('lease_break_period', type=int)
 
+    with self.argument_context('storage container-rm', resource_type=ResourceType.MGMT_STORAGE) as c:
+        from .sdkutil import get_container_access_type_names
+        c.argument('container_name', container_name_type, options_list=('--name', '-n'))
```

Comment:
Because container-rm is management plane operation, for each resource it will have resource id to use.
See id_part in document and try to apply on container.